searx/engines/mojeek.py

```diff
@@ -159,14 +159,23 @@ def fetch_traits(engine_traits: EngineTraits):
     from babel import Locale, UnknownLocaleError
     import contextlib
 
-    resp = network.get(base_url + "/preferences")
+    resp = network.get(base_url + "/preferences", headers={'Accept-Language': 'en-US,en;q=0.5'})
     dom = html.fromstring(resp.text)  # type: ignore
 
-    for code in eval_xpath_list(dom, f'//select[@name="{language_param}"]/option/@value'):
+    languages = eval_xpath_list(dom, f'//select[@name="{language_param}"]/option/@value')
+
+    engine_traits.custom['language_default'] = languages[0]
+
+    for code in languages[1:]:
         with contextlib.suppress(UnknownLocaleError):
-            Locale(code)
-            engine_traits.languages[code] = code
+            locale = Locale(code)
+            engine_traits.languages[locale.language] = code
+
+    regions = eval_xpath_list(dom, f'//select[@name="{region_param}"]/option/@value')
+
+    engine_traits.custom['region_default'] = regions[0]
+    engine_traits.custom['region_all'] = regions[1]
```

Comment:
Can't use `traits.all_locale` here 'cause that would be also be used by `traits.get_language`, which is undesirable as that value is only supported for regions.